### ISSUE TITLE
[Feature request] Add high contrast theme

### ISSUE THREAD

**Issue by beeradmoore** [beeradmoore/dlss-swapper#352]
**Is your feature request related to a problem? Please describe.**
High contrast theme was removed in [this](https://github.com/beeradmoore/dlss-swapper/commit/7f65dba06b884b108728f5402c782fc584116da7) commit. But because it is an accessibility feature, it should be re-added.

**Describe the solution you'd like**
Add it back in.

**Additional context**
Does high contrast have light/dark, do we just need to theme it?
